lib/taskana-core/src/main/java/pro/taskana/monitor/internal/MonitorMapperSqlProvider.java

```diff
@@ -25,13 +25,21 @@ public static String getTaskCountOfWorkbaskets() {
         + "SELECT T.WORKBASKET_KEY, (DAYS(T.${timestamp}) - DAYS(CAST(#{now} as TIMESTAMP))) "
         + "as AGE_IN_DAYS "
         + "</if> "
+        + "<if test=\"_databaseId == 'oracle'\">"
+        + "SELECT T.WORKBASKET_KEY, (TRUNC(T.${timestamp}) - TRUNC(CAST(#{now} as TIMESTAMP)))"
+        + " as AGE_IN_DAYS "
+        + "</if> "
         + "<if test=\"_databaseId == 'h2'\">"
         + "SELECT T.WORKBASKET_KEY, DATEDIFF('DAY', #{now}, T.${timestamp}) as AGE_IN_DAYS "
         + "</if> "
         + "<if test=\"_databaseId == 'postgres'\">"
         + "SELECT T.WORKBASKET_KEY, DATE_PART('DAY', T.${timestamp} - #{now}) as AGE_IN_DAYS "
         + "</if> "
-        + "FROM TASK AS T LEFT JOIN ATTACHMENT AS A ON T.ID = A.TASK_ID "
+        + "FROM TASK "
+        + "<if test=\"_databaseId != 'oracle'\">AS </if> "
```

Comment:
in other statements you've just removed the `AS` keyword. Do this here too? (applies for future if statements aswell ^^)

Reply:
DONE